### DISCUSSION TITLE
Next.js App requires 18.17 Node version, but encountering error with 16.20 Node version. how to resolve

### DISCUSSION THREAD

**Discussion by joseluis0320** [community/community#147343] [Apps, API and Webhooks] [answered]
### Select Topic Area

Question

### Body

When i create next js app using cli, an then i type

npm run dev 
I got this error.

You are using Node.js 16.20.2. For Next.js, Node.js version >= v18.17.0 is required.
But my node version is v20.10.0. How I fix this.

Solve this issue.

You are using Node.js 16.20.2. For Next.js, Node.js version >= v18.17.0 is required.

**Comment by dmark1029** [comment, marked as answer]
Update your Node.js installation to a version that meets or exceeds the required version (18.17.0). You can use a version manager like NVM (Node Version Manager) to easily switch between Node.js versions. Here are the steps to update using NVM:

nvm install 18.17.0
nvm use 18.17.0
then;

npm install again
npm run dev
